django_bucketav/settings.py

```diff
@@ -12,4 +12,4 @@
 
 
 def bucketav_models() -> "BucketAVModelDefinition":
-    return getattr(settings, "WAGTAIL_BUCKETAV_MODELS", {})
+    return getattr(settings, "DJANGO_BUCKETAV_MODELS", {})
```

Comment:
Suggestion: Given it's implied that settings are for Django, perhaps this should be `BUCKETAV_MODELS`?

Reply:
Good call! :slightly_smiling_face:  Handled in [b6d7f1c](https://github.com/torchbox/wagtail-bucketav/pull/6/commits/b6d7f1cca851c831b4a14ed8d86a41103598b23b)